Gemfile

```diff
@@ -65,37 +72,44 @@ gem 'docusign_rest', '~> 0.4.4'
 
 # We don't require Nokogiri directly, only through other gems (like axlsx).
 # However, due to CVE's, we need to ensure we're using a recent version.
-gem 'nokogiri', '~> 1.10.3'
+gem 'nokogiri', '>= 1.10.3'
 
 group :development, :test do
-  gem 'dotenv-rails', '~> 2.5.0'
+  gem 'dotenv-rails', '~> 2.8.0'
 
   # Testing
   gem 'bundler-audit', '~> 0.5'           # Linter
   gem 'capybara-screenshot'
   gem 'coffeelint', '~> 1.16'             # Coffeescript Linter
   gem 'database_cleaner', '~> 1.5'        # Truncates the DB after each test
-  gem 'factory_girl', '~> 4.7'            # Test object factories
-  gem 'faker', '~> 1.6'                   # Fake data generator
+  gem 'factory_bot', '~> 4.8'            # Test object factories
+  gem 'faker', '~> 3.2'                   # Fake data generator
   gem 'guard', '~> 2.14'                  # Continuous testing
   gem 'guard-minitest', '~> 2.4'          # ""
   gem 'letter_opener', '~> 1.7.0'         # Preview email on browser instead of sending
   gem 'm', '~> 1.5.0'                     # Allow to run individual tests
-  gem 'minitest', '~> 5.10.2'
-  gem 'minitest-around', '~> 0.4'         # Minitest around callback
-  gem 'minitest-rails-capybara', '~> 2.1' # Integration tests
+  gem 'minitest', '~> 5.14'
+  gem 'minitest-around', '~> 0.5'         # Minitest around callback
+  #gem 'minitest-rails-capybara', '~> 2.1' # Integration tests
   gem 'minitest-reporters', '~> 1.1'      # Test output format
+
+  gem 'minitest-rails', '~> 6.0'
+  #gem 'minitest-rails-capybara', '~> 3.0'
+  gem 'chosen-rails', '~> 1.10.0'
+  gem 'capybara'
+  gem 'rails-controller-testing'
+
   gem 'mocha', '~> 1.7.0'                 # Test mocking and stubbing
-  gem 'pry-byebug', '~> 3.6.0'            # Debugger
+  gem 'pry-byebug', '~> 3.9.0'            # Debugger
   gem 'pry-rails', '~> 0.3.5'
-  gem 'rack_session_access', '~> 0.1'     # Edit user-agent session
+  gem 'rack_session_access', '~> 0.2.0'     # Edit user-agent session
   gem 'reek', '~> 5.2'                    # Linter
   gem 'rubocop', '~> 0.60'                # Linter
```

Comment:
We like to use `standard` now.

Reply:
so another tasks is moving away from CAS and to OKTA, so this will be handled during that process.  Does that make sense?